packages/react-core/src/demos/Page/Page.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: Page
+id: Navigation
 section: components
 ---
```

Comment:
Move demos that include navigation into a new `demos/Nav/Nav.md` file with:
```
---
id: Navigation
section: components
---
```

Reply:
These are the demos that should be moved:
Default nav
Expandable nav
Horizontal nav
Tertiary nav
Grouped nav

Reply:
Thanks.